imap_processing/ultra/l2/l2_utils.py

```diff
@@ -0,0 +1,194 @@
+"""Build a solid angle map for a given spacing in degrees."""
+
+import typing
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+def build_solid_angle_map(
+    spacing: float, input_degrees: bool = False, output_degrees: bool = False
```

Comment:
A little nitpicky, but in the other code in geometry.py you had 

def cartesian_to_spherical(
    v: NDArray,
    degrees: bool = True,
    
 If possible be consistent.

Reply:
Good point! I prefer using radians, but had wanted to keep the default for the other function consistent with the existing code in degrees. I'll make this default to degrees as well and just override it